src/test/scala/io/iohk/ethereum/ObjectGenerators.scala

```diff
@@ -33,6 +33,8 @@ trait ObjectGenerators {
 
   def intGen(min: Int, max: Int): Gen[Int] = Gen.choose(min, max)
 
+  def posIntGen(min: Int, max: Int): Gen[Int] = Gen.choose(min, max).suchThat(_ > 0)
```

Comment:
why do we need this ? would `intGen(1, max: Int)` not work ? Or did it use 0 during shrinking ? 😞

Reply:
Oh I forgot about this issue. I don't know why but this test started failing consistently because it was generating negative numbers